README.md

```diff
@@ -7,10 +7,27 @@ Python support for Sublime's LSP plugin provided through [microsoft/pyright](htt
 1. Install [LSP](https://packagecontrol.io/packages/LSP) and
    [LSP-pyright](https://packagecontrol.io/packages/LSP-pyright) via Package Control.
 1. Restart Sublime.
+1. Configure pyright for your virtualenv.
```

Comment:
Why do we use number 1 for all list items? :)

Also, configuring virtualenv is an optional thing. It shouldn't be a part of installation.

Reply:
> Why do we use number 1 for all list items? :)

Sounds like you want to maintain those numbers manually?

> Also, configuring virtualenv is an optional thing. It shouldn't be a part of installation.

Though I may just add `(Optional)` to the list item. But I feel you want it to be a dedicated section?

Reply:
> Sounds like you want to maintain those numbers manually?

If we don't want to then let's just use `-`. Otherwise, we should make the text sensible both in text and rendered form.

> Though I may just add `(Optional)` to the list item.

`(Optional)` sounds good.

Reply:
> If we don't want to then let's just use -. Otherwise, we should make the text sensible both in text and rendered form.

I am fine as long as I am not going to be a maintainer of that. (And in fact, I don't think I am. :yum: )

Reply:
Maybe it's a convention in Markdown that I'm not aware of but I feel like the text should look reasonable both when read as "source" and "rendered". It's not that much to maintain in this case. ;))